### PR TITLE
Automate GitHub Release Draft

### PR DESCRIPTION
**Pull request by razo7**
Similar to NHC we add automation for the creation of Github Release upon Git tag creation.

[ECOPROJECT-1128](https://issues.redhat.com//browse/ECOPROJECT-1128)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -381,14 +381,18 @@ catalog-push: ## Push a catalog image.
...
+.PHONY: container-build-community
+container-build-community: docker-build bundle-build-community ## Build containers
 
 .PHONY: container-push 
 container-push: docker-push bundle-push catalog-build catalog-push ## Push containers (NOTE: catalog can't be build before bundle was pushed)
 
-.PHONY: container-build-and-push
-container-build-and-push: container-build container-push ## Build and push all the four images to quay (docker, bundle, and catalog).
+.PHONY: container-build-and-push-community
+container-build-and-push-community: container-build container-push ## Build three images, update CSV for community, and push all the images to Quay (docker, bundle, and catalog).
```
Comment (mshitrit) [inline Makefile:395]: IIUC instead of `container-build` it should be `container-build-community`

File: Makefile [hunk 2]
```diff
@@ -381,14 +381,18 @@ catalog-push: ## Push a catalog image.
 verify-unchanged: ## Verify there are no un-committed changes
 	./hack/verify-unchanged.sh
 
-.PHONY: container-build
```
Comment (mshitrit) [inline Makefile:384]: Don't we want to keep `container-build` for non community builds ?

Comment (razo7) [reply Makefile:384]: Yes, why not :)

File: .github/workflows/post-submit.yaml [hunk 3]
```diff
@@ -25,9 +29,35 @@ jobs:
...
+        uses: meeDamian/github-release@2.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          draft: true
+          body: |
+            # Fence Agents Remediation ${{ github.ref_name }}
+            
+            ## Notable Changes
+            
+            * TODO
```
Comment (slintes) [inline .github/workflows/post-submit.yaml:51]: nit: you might want to have an emptly line after this

**Pull request merged**